accounts/models.py

```diff
@@ -10,3 +11,8 @@ class Meta:
 
     def __str__(self):
         return self.username
+
+    #store user preferences in a one to one mapping
+    preferences = models.OneToOneField(Preferences, on_delete=models.CASCADE, blank=True, null=True) 
```

Comment:
Let’s define the recon the Preferences model rather than the user. Then, use a ‘reverse_name=“preferences”’ on the ‘user’ OneToOne field attached to the Preferences model. That way, we assign the user to the Preferences instance directly in view.py.

Reply:
```suggestion
```